fullstack/src/app/page.js

```diff
@@ -5,38 +5,19 @@ import LineChartComponent from "@/components/graphs/LineChartComponent";
 import BarChartComponent from "@/components/graphs/BarChartComponent";
 import "./page.css";
 
-// Dummy data for the graphs
-const temperatureData = [
-  { day: 'Monday', temperature: 20 },
-  { day: 'Tuesday', temperature: 22 },
-  { day: 'Wednesday', temperature: 19 },
-  { day: 'Thursday', temperature: 24 },
-  { day: 'Friday', temperature: 25 },
-  { day: 'Saturday', temperature: 23 },
-  { day: 'Sunday', temperature: 21 },
-];
-
-const airPressureData = [
-  { day: 'Monday', pressure: 1000 },
-  { day: 'Tuesday', pressure: 1005 },
-  { day: 'Wednesday', pressure: 1010 },
-  { day: 'Thursday', pressure: 1008 },
-  { day: 'Friday', pressure: 1007 },
-  { day: 'Saturday', pressure: 2000 },
-  { day: 'Sunday', pressure: 1002 },
-];
-
 export default function Home() {
   return (
     <div className="app-container">
       <div className="widgets">
-        <Widget name={'Temperature'} data={temperatureData} GraphComponent={LineChartComponent} datakey="temperature"/>
-        <Widget name={'Air Pressure'} data={airPressureData} GraphComponent={BarChartComponent} datakey="pressure"/>
-        <Widget name={'Humidity'} data={'coming soon'} GraphComponent={null} />
-        <Widget name={'Wind'} data={'coming soon'} GraphComponent={null} />
+        <Widget name="Temperature" dataKey="temperature" GraphComponent={null} />
+        <Widget name="Air Pressure" dataKey="pressure" GraphComponent={null} />
+        <Widget name="Humidity" dataKey="humidity" GraphComponent={null} />
+        <Widget name="Wind" dataKey="wind" GraphComponent={null} />
+        <Widget name="CO2" dataKey="co2" GraphComponent={null} />
+        <Widget name="Gas" dataKey="gas" GraphComponent={null} />
+        <Widget name="Dust" dataKey="dust" GraphComponent={null} />
```

Comment:
This is missing rain but I understand it is also not in the routes. We can add this next merge.